mne/viz/epochs.py

```diff
@@ -1967,7 +1967,7 @@ def _plot_histogram(params):
             ax.plot((rej, rej), (0, ax.get_ylim()[1]), color='r')
         plt.title(titles[types[idx]])
     params['histogram'].suptitle('Peak-to-peak histogram', y=0.99)
-    params['histogram'].subplots_adjust(hspace=0.6)
+    params['histogram'].subplots_adjust(hspace=1)
```

Comment:
why not a call to tight_layout?

Reply:
'cos it messes up with the title anyway
![image](https://user-images.githubusercontent.com/7044835/40728567-aee9bb06-642a-11e8-8582-ee07dca459be.png)